libs/service-portal/settings/src/components/Forms/PhoneForm/Steps/ConfirmationStep.tsx

```diff
@@ -68,28 +68,37 @@ export const ConfirmationStep: FC<Props> = ({
           )}
         />
       </Box>
-      <Box display="flex" justifyContent="spaceBetween" marginTop={4}>
-        <Button variant="ghost" onClick={onBack}>
-          {formatMessage({
-            id: 'service.portal:go-back',
-            defaultMessage: 'Til baka',
-          })}
-        </Button>
-        <Button
-          type="submit"
-          variant="primary"
-          icon="arrowForward"
-          disabled={loading}
-        >
-          {formatMessage(
-            submitButtonText
-              ? submitButtonText
-              : {
-                  id: 'sp.settings:save-changes',
-                  defaultMessage: 'Vista breytingar',
-                },
-          )}
-        </Button>
+      <Box
+        display="flex"
+        justifyContent="spaceBetween"
+        flexWrap="wrap"
+        marginTop={4}
+      >
+        <Box marginBottom={[1, 0]}>
+          <Button variant="ghost" onClick={onBack}>
+            {formatMessage({
+              id: 'service.portal:go-back',
```

Comment:
Shouldn't it be "sp" instead of "service.portal"

Reply:
The service.portal namespace is used for system strings which are used by more than one libaries